WordPress/WordPressUITests/Utils/XCTest+Extensions.swift

```diff
@@ -26,13 +26,33 @@ extension XCUIElement {
     }
 }
 
-extension XCTest {
-
-    func isIPhone() -> Bool {
-        return UIDevice.current.userInterfaceIdiom == .phone
-    }
+var isIPhone: Bool {
+    return UIDevice.current.userInterfaceIdiom == .phone
 }
 
-func isIpad() -> Bool {
+var isIpad: Bool {
```

Comment:
I've switched this to be a variable rather than a function, because it takes no arguments (and isn't likely to any time soon)